contracts/Colony.sol

```diff
@@ -31,27 +31,27 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
   function setFounderRole(address _user) public stoppable auth {
     // To allow only one address to have founder role at a time, we have to remove current founder from their role
     ColonyAuthority colonyAuthority = ColonyAuthority(address(authority));
-    colonyAuthority.setUserRole(msg.sender, FOUNDER_ROLE, false);
-    colonyAuthority.setUserRole(_user, FOUNDER_ROLE, true);
+    colonyAuthority.setUserRole(msg.sender, uint8(ColonyRole.Founder), false);
+    colonyAuthority.setUserRole(_user, uint8(ColonyRole.Founder), true);
 
     emit ColonyFounderRoleSet(msg.sender, _user);
   }
 
   function setAdminRole(address _user) public stoppable auth {
-    ColonyAuthority(address(authority)).setUserRole(_user, ADMIN_ROLE, true);
+    ColonyAuthority(address(authority)).setUserRole(_user, uint8(ColonyRole.Admin), true);
```

Comment:
Can these functions take `ColonyRole` enums directly? Do we have to cast?

Reply:
Yes unfortunately `enum`s don't convert implicitly

Reply:
Right but can `setUserRole` be updated to accept the `ColonyRole` type?

Reply:
`setUserRole` is part of `DSRoles` which works generically with `uint8`s and so we can't change that.